beetsplug/badfiles.py

```diff
@@ -30,6 +30,16 @@
 import six
 
 
+class CheckerCommandException(Exception):
+    """Raised when running a Checker failed."""
```

Comment:
Could you please expand the docstring just a little to explain what the fields mean (`checker`, `path`, `errno`, and `msg`)?